src/Common/Commands.Common/AzurePSCmdlet.cs

```diff
@@ -66,6 +66,19 @@ protected override void BeginProcessing()
             base.BeginProcessing();
         }
 
+        /// <summary>
```

Comment:
Well, my code actually needs this. Could you please remove it when you add the similar functionality?